src/components/lab-callout/index.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image'
+import Button from 'components/button'
+import { useInstruqtEmbed } from 'contexts/instruqt-lab'
+import s from './interactive-lab-callout.module.css'
+
+export default function InteractiveLabCallout() {
+  const ctx = useInstruqtEmbed()
+
+  if (!ctx.labId) {
+    return null
+  }
+
+  return (
+    <div className={s.interactiveCallout}>
+      <div>
+        <h2 className={s.title}>Launch Terminal</h2>
+        <p className={s.description}>
+          This tutorial includes a free interactive command-line lab that lets
+          you follow along on actual cloud infrastructure.
+        </p>
+        <div className={s.ctaButton}>
+          <Button
```

Comment:
I think this needs the `type="secondary"` prop.

Reply:
✅ made this `"secondary"` - i think this addresses @kendallstrautman 's question about the blue button below as well (talk about me missing basic stuff from the design spec 😅 sorry about that!)